commit 72ed254b38571a2139ed2a43607e07750fea459f
Author: Jim Fehlig <jfehlig@suse.com>
Date:   Thu Jan 30 11:29:41 2020 -0700

    build: Drop execute permissions on virsh bash completion command
    
    rpmlint complains about "script-without-shebang" due to the execute
    permissions on /usr/share/bash-completion/completions/vsh. Use
    INSTALL_DATA instead of INSTALL_SCRIPT to avoid the unnecessary
    execute permissions.
    
    Signed-off-by: Jim Fehlig <jfehlig@suse.com>
    Reviewed-by: Michal Privoznik <mprivozn@redhat.com>

Index: libvirt-6.0.0/tools/Makefile.am
===================================================================
--- libvirt-6.0.0.orig/tools/Makefile.am
+++ libvirt-6.0.0/tools/Makefile.am
@@ -357,7 +357,7 @@ libvirt-guests.service: libvirt-guests.s
 if WITH_BASH_COMPLETION
 install-bash-completion:
 	$(MKDIR_P) "$(DESTDIR)$(BASH_COMPLETIONS_DIR)"
-	$(INSTALL_SCRIPT) $(srcdir)/bash-completion/vsh \
+	$(INSTALL_DATA) $(srcdir)/bash-completion/vsh \
 		"$(DESTDIR)$(BASH_COMPLETIONS_DIR)/vsh"
 	( cd $(DESTDIR)$(BASH_COMPLETIONS_DIR) && \
 		rm -f virsh virt-admin && \
